### PR TITLE
updated iin

### PR DESCRIPTION
**Pull request by ankitdas13**
Api doc reference

[Fetch All IINs Supporting Native OTP](https://razorpay.com/docs/api/payments/cards/iin-api/#fetch-all-iins-supporting-native-otp)

[Fetch All IINs with Business Sub-type](https://razorpay.com/docs/api/payments/cards/iin-api/#fetch-all-iins-with-business-sub-type)

### CODE REVIEW

File: src/test/java/com/razorpay/IinClientTest.java [hunk 1]
```diff
@@ -54,4 +56,27 @@ public void fetch() throws RazorpayException {
...
+    @Test
+    public void fetchList() throws RazorpayException {
+
+        JSONObject request = new JSONObject();
+        request.put("flow","otp");
+
+        JSONObject mockedResponseJson = new JSONObject();
+        mockedResponseJson.put("count", 0);
+        mockedResponseJson.put("entity", "iin");
+        JSONArray iinArr = new JSONArray();
```
Comment (rohitcbr) [inline src/test/java/com/razorpay/IinClientTest.java:69]: make it iinArray instead for better readability

Comment (ankitdas13) [reply src/test/java/com/razorpay/IinClientTest.java]: made typo correction

Comment (ankitdas13) [reply src/test/java/com/razorpay/IinClientTest.java]: https://github.com/razorpay/razorpay-java/blob/020d762e5d69323acbd579dabe42a79e2844de4f/src/test/java/com/razorpay/IinClientTest.java#L69

Comment (rohitcbr) [reply src/test/java/com/razorpay/IinClientTest.java]: +1

**Review by rohitcbr — APPROVED** [review]

**Pull request merged**